src/current/v24.1/plpgsql.md

```diff
@@ -254,7 +261,9 @@ WHILE condition LOOP
 
 For an example, see [Create a stored procedure that uses a `WHILE` loop]({% link {{ page.version.version }}/create-procedure.md %}#create-a-stored-procedure-that-uses-a-while-loop).
 
-### `EXIT` and `CONTINUE` statements
+### Add exit and continue logic
```

Comment:
[nit] This heading seems a little odd when we have a RETURN section in between the EXIT and CONTINUE examples.

Reply:
How does "Control execution flow" sound to you?

Reply:
That SGTM.